src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+
+}
+
+body {
+  margin: 0;
+  padding-left: 40px;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.search-bar__input {
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  flex-shrink: 0;
+  background-color: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search-bar__input--big {
+  padding: 27px 0 26px 61px;
+  background: url("./images/Search.svg") 25px 24px no-repeat;
+  background-size: 19px;
+}
+
+.search-bar__input--small {
+  padding: 12px 0 13px 33px;
+  background: url("./images/Search.svg") 12px 14px no-repeat;
+  background-size: 11px;
+}
+
+.search-bar__input::placeholder {
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+  font-weight: 300;
```

Comment:
This is the default font-weight

Reply:
there is a requirement to set default font weight in readme.md, i thought it says i should specify font weight like this.
Should I remove font-weight ? because i don't complete understand what you ask me to do in this comment.

![req1](https://github.com/mate-academy/layout_search-bar-airbnb/assets/113614842/8196310e-1b74-4c2a-9219-73623941c882)